bpf/lib/host_firewall.h

```diff
@@ -125,9 +124,10 @@ __ipv6_host_policy_egress(struct __ctx_buff *ctx, bool is_host_id __maybe_unused
 
 	/* Only create CT entry for accepted connections */
 	if (ret == CT_NEW && verdict == CTX_ACT_OK) {
+		struct ct_state ct_state_new = {};
```

Comment:
or
```
struct ct_state ct_state_new = {
        .src_sec_id = HOST_ID,
        .proxy_redirect = proxy_port > 0,
};
```

Reply:
:nod: but it's not a pattern we typically follow I think. Luckily makes no difference to the compiler.